drivers/volume/portworx/portworx.go

```diff
@@ -1453,6 +1454,77 @@ func (d *portworx) ValidatePureFaFbMountOptions(volumeName string, mountoption [
 
 }
 
+func (d *portworx) ValidatePureFaCreateOptions(volumeName string, FStype string, volumeNode *node.Node) error {
```

Comment:
please add a comment line before this function 
// ValidatePureFaCreateOptions returns <what function does>

Reply:
Done.